src/background.js

```diff
@@ -42,16 +42,18 @@ function init() {
   log('init()');
   browser.runtime.onInstalled.addListener((details) => {
     if (details.reason === 'install') {
-        const title = browser.i18n.getMessage('bookmarkFolderTitle');
-        browser.bookmarks.search({title: title}).then(folders => {
+      getMetricsUUID().then(clientUUID => {
```

Comment:
I'd say let's not use the metrics UUID for this because it's potentially close to PII, but all I have is vague handwaving and no real scenarios of doom.